.builders/images/linux-aarch64/Dockerfile

```diff
@@ -19,8 +19,8 @@ ENV LDFLAGS="-Wl,-rpath,'\$\$ORIGIN' -Wl,--strip-debug"
 RUN yum install -y perl-IPC-Cmd perl-CPANPLUS perl-core && \
  cpanp -i List::Util 1.66 && \
  DOWNLOAD_URL="https://www.openssl.org/source/openssl-{{version}}.tar.gz" \
- VERSION="3.5.2" \
- SHA256="c53a47e5e441c930c3928cf7bf6fb00e5d129b630e0aa873b08258656e7345ec" \
+ VERSION="3.5.4" \
```

Comment:
I find it disturbing that we build openssl in a different way between integrations-core and the main agent. I'ld love to find out more, towards resolving the differences, but that is beyond the scope of this PR.